src/targetNumber.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const targetNumber = Math.round(Math.random() * 9000 + 999).toString();
+
+module.exports = { targetNumber };
```

Comment:
Better export a function to generate a number, not the number itself, as per requirements:

> Please create separate modules for **generating a number**...